src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -484,6 +492,14 @@ public virtual bool IsLambdaParam(int index)
             return _maskLambdas.TestBit(index);
         }
 
+        // Returns true if the parameter at the specified 0-based rank is an identifier, false otherwise.
+        public virtual bool IsIdentifier(int index)
+        {
+            Contracts.AssertIndexInclusive(index, MaxArity);
+
+            return _maskIdentifiers.TestBit(index);
```

Comment:
>_maskIdentifiers [](http://example.com/codeflow?start=19&length=16)

If this is virtual, can we just skip _maskIdentifiers altogether and defer to the derived function?

We'll need that anyways for varargs like AddColumns.

Reply:
Actually, this would also be very helpful in significantly reducing the code churn.

Reply:
Found an alternative